### PR TITLE
fix: add peer review requirments section

### PR DESCRIPTION
**Pull request by bradleycamacho**
I created a table with a few ideas of what constitutes a PR that does and doesn't need to be reviewed. Please leave any feedback or ideas you have!

### CODE REVIEW

File: src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx [hunk 1]
```diff
@@ -7,6 +7,49 @@ topics:
...
+</th>
+<th>
+Needs a peer review
+</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>
+Edits less than 5 lines.
```
Comment (barbnewrelic) [inline src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx:28]: if you want to be grammatically correct this s/be "fewer" but that always sounds weird

File: src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx [hunk 2]
```diff
@@ -7,6 +7,49 @@ topics:
...
+</td>
+<td>
+Edits more than 5 lines.
+</td>
+</tr>
+<tr>
+<td>
+Formatting and small markdown fixes such as fixing links, making a UI component bold, putting a command in a code block, etc.
```
Comment (barbnewrelic) [inline src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx:36]: I think "markdown" s/be capital M but you should probably double check

**Review by barbnewrelic — COMMENTED** [review]
This is a good start! I'll wait until the build finishes so I can see what this table actually looks like, then may want to add more thoughts.

File: src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx [hunk 3]
```diff
@@ -7,6 +7,49 @@ topics:
 
 Use this document to learn how to review and peer edit docs for you fellow writers in GitHub. Check the [Tech Writer workflow doc](/docs/style-guide/writer-workflow/tech-writer-workflow/) for info on how to set up your local environment. To troubleshoot GitHub issues, see [our guide](/docs/style-guide/writer-workflow/github-troubleshooting/).
 
+## What needs a peer review [#peer-reqs]
+
+Any substantial pull request should be peer reviewed. The line between substantial and trivial is a bit blurry, so refer to the following table when in doubt:
```
Comment (barbnewrelic) [inline src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx:12]: Team meeting notes recommend any convos about review should take place in the PR itself as much as possible. IIRC we also agreed that sometimes convos need to take place in slack, so try to summarize that convo for public consumption in the PR. Recommend that you add something about this to style guide.

Comment (mmfred) [inline src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx:12]: I would bold or otherwise emphasize that first sentence about Any substantial pull request.

File: src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx [hunk 4]
```diff
@@ -7,6 +7,49 @@ topics:
...
+
+<table>
```
Comment (barbnewrelic) [inline src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx:14]: As noted in slack convo 11-10-2021, this table will needs some adjustments. :-)

File: src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx [hunk 5]
```diff
@@ -7,6 +7,49 @@ topics:
...
+<td>
+Collapser and table edits.
+</td>
+</tr>
+<tr>
+<td>
+Redirects not related to recent taxonomy changes.
+</td>
+<td>
+Any changes that affect the taxonomy and redirects that result from those changes.
```
Comment (mmfred) [inline src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx:47]: On first reading, I thought this was referring to the "recent taxonomy changes" so maybe reword: Any taxonomy changes and redirects that result from those changes.

File: src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx [hunk 6]
```diff
@@ -7,6 +7,49 @@ topics:
...
+</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>
+Edits less than 5 lines.
+</td>
+<td>
+Edits more than 5 lines.
```
Comment (barbnewrelic) [inline src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx:31]: I disagree with 5 lines. I think more than 10-15 lines would need a PR.

File: src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx [hunk 7]
```diff
@@ -7,6 +7,49 @@ topics:
...
+</td>
+</tr>
+<tr>
+<td>
+Formatting and small markdown fixes such as fixing links, making a UI component bold, putting a command in a code block, etc.
+</td>
+<td>
+Collapser and table edits.
```
Comment (mmfred) [inline src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx:39]:
```suggestion
Collapser and table edits. This applies even if you don't change the content, but just format into or out of collapsers or tables.
```

File: src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx (hunk 6 above)
Comment (barbnewrelic) [reply src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx:31]: I also wonder if we need some guidelines around number of pages that an edit touched? For example, Both zulu and I apply the exact same text to multiple docs from time to time. The actual text edit may be minimal, but the number of docs may be a lot. I honestly don't think a one line changed applied to 25 docs does not need a peer review, but am interested in other writers' thoughts.

File: src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx [hunk 8]
```diff
@@ -7,6 +7,49 @@ topics:
...
+<td>
+Redirects not related to recent taxonomy changes.
+</td>
+<td>
+Any changes that affect the taxonomy and redirects that result from those changes.
+</td>
+</tr>
+</tbody>
+</table>
+
```
Comment (barbnewrelic) [inline src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx:52]: One other thing not addressed in this draft was brought up in team meeting: Amount of time for the build to build, plus the amount of time to get a review, plus the amount of time to review. This all adds up to significant time delays, which is one reason why I think our Jira board has been affected by the ability to complete the work. If a PR affects a time-sensitive release and it affects multiple docs, and the Gatsby build works, I am in favor of proceeding without a peer review. But I don't think we reached consensus about the time impact.

**Review by barbnewrelic — CHANGES_REQUESTED** [review]
Recommend do not publish this PR until table is fixed and all TWs weigh in with their thoughts.

File: src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx (hunk 8 above)
Comment (bradleycamacho) [reply src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx:52]: I think we can use our best judgement in situations like this! One thing @zuluecho9 does is ping me specifically when I'm assigned one of his PRs and he needs to get them out in the next day or so.

Comment (bradleycamacho) [reply src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx:52]: Basically, I think we can be really active about telling our reviewers when something is a priority or not. If it is a priority and our reviewer is busy, we can just ask the team if anyone is available to review. @austin-schaefer does this all the time.

File: src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx (hunk 6 above)
Comment (urbiz-nr) [reply src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx:31]: I say go with your gut. I (almost) always check stuff locally, that's why I'm confident of the changes.

Comment (paperclypse) [reply src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx:31]: I always check stuff locally or in the cloud preview build. This is super important, I think.

Comment (paperclypse) [reply src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx:31]: @barbnewrelic What if that one line applied to 25 docs could be slightly improved by a second brain thinking about it? Then you've improved 25 docs :) 

There are extremely minor edits that I've made that have been improved by another writer taking a couple minutes to think about it.

File: src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx (hunk 2 above)
Comment (paperclypse) [reply src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx:36]: Yeah, Markdown is capitalized

File: src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx (hunk 6 above)
Comment (bradleycamacho) [reply src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx:31]: Yeah, I agree with @paperclypse here. Another point is, if it's just a small change, does it really matter if it takes an extra day or two for someone to review it? 

I do think we can be really flexible with where the line is drawn here though.

File: src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx (hunk 8 above)
Comment (paperclypse) [reply src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx:52]: I think it would be helpful to clarify what we mean when we say "PR review" and what we mean when we say "peer edit". I think these things have gotten a bit muddied.

To my mind: 
A PR review is for non-Jira PRs that need a second pair of eyes. I feel like it's ok for this to be a pretty quick pass.
A peer edit is related to a specific Jira and should probably be a bit more in-depth.

File: src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx (hunk 6 above)
Comment (bradleycamacho) [reply src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx:31]: And for the record, I think this 5 line limit is more so for reviewing the content itself than a fear that a change more than 5 lines could break something.

File: src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx (hunk 4 above)
Comment (bradleycamacho) [reply src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx:14]: 100%!

I'll get an updated version of this table pushed up once we're a bit more sure about the requirements.

File: src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx (hunk 8 above)
Comment (bradleycamacho) [reply src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx:52]: @paperclypse I really like this distinction.

**Comment by barbnewrelic** [comment]
We have all raised really good points, but I don't think we are coming to a
consensus yet about how to resolve our different points of view about, can
a writer trust his/her own judgment, esp. about small edits? And about
timing, maybe a "small" change really is time-critical? I personally would
prefer a completely different approach: If I know I'm going to make a small
edit but it may or may not affect 25 docs, I would rather do the peer
review BEFORE I begin. Propose my change to another writer, get his/her
feedback before I begin, then I apply my change to all the kazillion docs.
I'm still responsible for making sure all those docs in my PR
build correctly, but I don't need a reviewer when I'm ready to merge
because I've already had the sanity check. That has the added benefit of,
another writer does not need to set aside time in their own (already busy)
workday to review my kazillion docs, minor edit or not.
